server/routes.js

```diff
@@ -16,7 +16,7 @@ export default function(server) {
   });
 
   server.get(/^\/(images|fonts)\/.*/, function(req, res) {
```

Comment:
For performance reasons the `301` redirect should be done in web server level (ie: `apache`/`nginx`). Where these required can be cached for ling time will lead to better performance on the application layer.